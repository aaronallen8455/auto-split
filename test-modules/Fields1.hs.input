main :: IO ()
main = pure ()

data R = R { r1 :: Int, r2 :: Bool, r3 :: Bool }

test :: R
test = FIELDS R { r1 = 1 }
