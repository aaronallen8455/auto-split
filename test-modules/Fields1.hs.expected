main :: IO ()
main = pure ()

data R = R { r1 :: Int, r2 :: Bool, r3 :: Bool }

test :: R
test = R { r1 = 1, r2 = _, r3 = _ }
